.github/workflows/release.yml

```diff
@@ -69,6 +78,44 @@ jobs:
             github.rest.git.createRef({
               owner: context.repo.owner,
               repo: context.repo.repo,
-              ref: 'refs/tags/lambda-v' + "${{ env.VERSION }}",
+              ref: 'refs/tags/' + "${{ steps.version.outputs.tag }}",
               sha: context.sha
             })
+
+  regular-sar:
```

Comment:
I agree with this.

We also only upload dependencies to the bucket when this workflow succeeds:

https://github.com/elastic/elastic-serverless-forwarder/blob/70a2ca89087f664d3d2a14699b6aa0856a620f03/.github/workflows/upload-dependencies.yml#L11-L15

Reply:
IIUC, there is a corner case in https://github.com/elastic/elastic-serverless-forwarder/blob/70a2ca89087f664d3d2a14699b6aa0856a620f03/.github/workflows/upload-dependencies.yml#L32 and it could use a wrong version if `2 or more` commits happened in a row in `share/version.py` while `upload-dependencies.yml` has not been executed yet.

We can do a follow-up and see if that's a genuine reason

Reply:
I agree with you. There is also another problem: if a commit happens and the tag is not updated (so no version upgrade). I don't see why a commit would not cause a version upgrade, but we do not check that.

Reply:
Good catch!